models/balances/optimism/erc20/balances_optimism_erc20_latest.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(
+        tags = ['dunesql'],
+        alias = alias('latest'),
+        post_hook='{{ expose_spells(\'["optimism"]\',
```

Comment:
It might make sense to materialize the `latest` tables as they might end up being used in a lot of queries/dashboards/models downstream? Thoughts @jeff-dude?

Reply:
a view is nice for most up-to-date data, but if the performance is poor, looking to make incremental is worthwhile to try